includes/category_featured_images_widget.php

```diff
@@ -17,7 +17,12 @@ class RSS_Post_Aggregation_Category_Featured_Images_Widget extends WP_Widget {
 		'read_more_text' => '',
 	);
 
-	private $rss_args = array (
+	/**
+	 * RSS Args.
+	 *
+	 * @var array
```

Comment:
@kailanwyatt let's go through all the PHP code and add `@since 0.1.1` ( current version ) - I'd like to run API-gen on this at a later date.